core/common/src/main/java/alluxio/worker/block/io/LocalFileBlockWriter.java

```diff
@@ -34,6 +35,8 @@
   private final RandomAccessFile mLocalFile;
   private final FileChannel mLocalFileChannel;
   private final Closer mCloser = Closer.create();
+  private long mPosition;
+  private boolean mClosed;
```

Comment:
Every class member is set to default implicitly. I tried not to initialize anything to default unnecessarily (similarly, we don't initialize class member objects to null). https://docs.oracle.com/javase/specs/jls/se8/html/jls-4.html#jls-4.12.5